modules/web/src/com/haulmont/sampler/web/ui/components/radiobuttongroup/description/radiobuttongroup-option-description.xml

```diff
@@ -0,0 +1,6 @@
+<window xmlns="http://schemas.haulmont.com/cuba/screen/window.xsd">
+    <layout>
+        <radioButtonGroup id="radioButtonGroup"
```

Comment:
It seems that a caption is required. Currently, it's unclear what values we see on the demo.